tools/automator-main.sh

```diff
@@ -270,9 +282,15 @@ merge() {
   local src_branch="${AUTOMATOR_SRC_BRANCH:-none}"
   fork_name="$src_branch-$branch-$modifier-$(hash "$title")"
   git remote add -f -t "$merge_branch" upstream "$merge_repository"
+  local code
   set +e # git return a non-zero exit code on merge failure, which fails the script
-  git -c "user.name=$user" -c "user.email=$email" merge --no-ff -m "$title" --log upstream/"$merge_branch"
-  local code=$?
+  if [ "${merge_strategy}" == "merge" ]; then
+    git -c "user.name=$user" -c "user.email=$email" merge --no-ff -m "$title" --log upstream/"$merge_branch"
+    code=$?
+  else
+    git -c "user.name=$user" -c "user.email=$email" rebase upstream/"$merge_branch"
+    code=$?
```

Comment:
This line is duplicated across the if/else.. I wonder: would it work if we would have just a single one below the `fi`?
(not sure, but intuitively I would assume that closing the control flow will not change the last exit code)?

Reply:
Yeah, fixed. This will need to change in the future if we add other commands after git. But it's safe for now. Thanks.